contracts/adapters/Ragequit.sol

```diff
@@ -67,6 +67,12 @@ contract RagequitContract is IRagequit, DaoConstants, MemberGuard {
         uint256 lootToBurn,
         address[] memory tokens
     ) external override {
+        // Checks if the are enough shares and/or loot to burn
+        require(sharesToBurn + lootToBurn > 0, "insufficient shares/loot");
```

Comment:
This check was added to ensure we fail the call right the member is triggering it with 0 shares and 0 loot.